pyat/at/physics/dynamic_aperture.py

```diff
@@ -0,0 +1,895 @@
+import at
+import copy
+import at.plot
+from itertools import compress
+import numpy as np
+import math
+import pickle
+from scipy.io import savemat
+import time
+
+__all__ = ['Acceptance6D', 'dynamic_aperture', 'off_energy_dynamic_aperture', 'momentum_acceptance']
+
+
+class Acceptance6D(object):
+
+    verbose = True
+
+    # possible acceptance planes
+    planes = ('x', 'xp', 'y', 'yp', 'delta', 'ct')
+
+    # 2D combination of planes
+    modes = ('x-y',
+             'delta-x',
+             '6D',
+             'xp-yp',
+             'x-xp',
+             'y-yp',
+             'ct-delta')
```

Comment:
Apparently it's nor used anywhere

Reply:
now it is at least in one place! I added it to use it for user input check.